docker/testnet/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3"
+services:
+  bitcoind:
+    container_name: bitcoind
+    privileged: true
+    image: oneforalonee/bitcoind:v26.0
+    restart: on-failure
+    hostname: bitcoind
+    stop_grace_period: 5m
+    environment:
+      USERID: ${USERID:-1000}
+      GROUPID: ${GROUPID:-1000}
+    command: >-
+      bitcoind -server=1 -testnet=1
+      -zmqpubrawblock=tcp://0.0.0.0:28334
+      -zmqpubrawtx=tcp://0.0.0.0:28335 -zmqpubhashblock=tcp://0.0.0.0:28336
+      -txindex=1 -rpcbind=0.0.0.0 -rpcallowip=0.0.0.0/0
+      -rpcport=18332 -rest -listen=1 -listenonion=0
+      -rpcauth=alice:128f22494cc2208ea8376a3d0b45a131$9cc0187c0e49f35454a3ed1250e40ecaef420cfcd294d3ac22496adbe64f04b9
```

Comment:
Is it possible to not use a constant `rpcauth`?

Reply:
I'm using this constant `rpcauth` just for easy use. Others may delete it and generate their own `rpcauth` via [rpcauth.py](https://github.com/bitcoin/bitcoin/tree/master/share/rpcauth) and put it to `./bitcoind/bitcoin.conf` and change the line `auth = "alice:alice"  in `conf.toml` to their own config.
Also, if you want to using cookie file to auth, there's a way to do it:
1. you can remove the `rpcauth` config
2. append one line in `docker-compose.yml` in `[electrs] > volumes` section: `- ./cookie:/electrs/.cookie`
3. adjust the `conf.toml` file, i.e. comment out the `rpcauth` line, un-comment `cookie_file` and set it's value to `/electrs/.cookie`
4. start with `docker compose up -d`, this command will only start bitcoind, but not electrs
5. copy the bitcoind's cookie file's content to `./cookie` file: `cat ./bitcoind/testnet3/.cookie > cookie`
6. rebuild electrs with `docker compose up -d --build --no-deps`
But i thought it's kind complicate, most people just want one-click to use it.